po/es.po

```diff
@@ -22,7 +22,7 @@ msgstr ""
 #: data/dev.tchx84.Portfolio.metainfo.xml.in:11 src/window.ui:296
 #: src/window.ui:508 src/window.ui:1889
 msgid "Portfolio"
-msgstr ""
+msgstr "Portafolio"
```

Comment:
I don't think we should translate the name of the app, since it's not really that generic to be understood in non-english.